background.ts

```diff
@@ -160,6 +158,16 @@ Electron.app.whenReady().then(async() => {
       await window.openPathUpdate();
     }
 
+    try {
+      await removeLegacySymlinks(paths.oldIntegration);
+    } catch (error) {
+      if (error instanceof PermissionError) {
+        await window.openLegacyIntegrations();
```

Comment:
I'm wondering if we want to alert each time rancher desktop is opened if the user doesn't take any action to clean things up. What do you think?

Reply:
We could add a checkbox that lets people say "don't show me this again". Which would be more work, and an addition to the settings object.

On the other hand, this popup occurs when somehow the user has manually created symlinks in `/usr/local/bin`, and `/usr/local/bin` is not owned by their user. This isn't going to be a common thing.

In that light, I'd argue that this isn't worth the time/complexity cost. But I'm open to ideas as always :)

Reply:
I think we're fine without the checkbox as long as it's clear what action the user needs to take in order to get rid of the prompt.